### DISCUSSION TITLE
Can I use groups only for relative positioning the items inside it ?

### DISCUSSION THREAD

**Discussion by ghost** [fabricjs/fabric.js#8473] [Q&A] [answered]
I have bigger canvas  and inside that there is drawable area where I can put objects. I want to make that drawable area a Group and whenever I add objects to that group, those objects should be relatively positioned to the group. This is working but the problem is that, if I click on any of the objects inside that group, all the objects (or the group) gets selected. But I don't want this behavior. I just want to use the Group only for relatively positioning items inside it. Is it possible somehow ?

[https://jsbin.com/gabopet/9/edit?html,css,js,console,output](url)

**Comment by iocoker** [comment, marked as answer]
If what you want to achieve is a drawable area, so anything outside should be clipped off, you can use a clipPath on your canvas instead. So, you say `canvas.clipPath = r1`.

But if you want a group and will like the group to be moved, you can try the new [Group in v6](https://github.com/fabricjs/fabric.js/issues/7670)
[Demo](http://jsfiddle.net/qxdk4h9L/2/)

**Reply by ShaMan123** [reply]
Good suggestion for <v6!

**Reply by ghost** [reply]
@iocoker Thanks for the clipPath idea and suggesting v6. But is there any workaround for relative positioning items/objects in <v6 ?
What you see in the provided jsfiddle is that if you click any of the circle both the circles are getting selected as both are grouped. But this is not what I wanted. I wanted the Group to just work for relative positioning the items.

**Comment by ShaMan123** [comment]
v6 is what you are after.
What @iocoker suggests is a good workaround.
